packages/aws-cdk-lib/aws-events/test/event-bus.test.ts

```diff
@@ -281,6 +283,19 @@ describe('event bus', () => {
     }).toThrow(/'eventSourceName' must satisfy: /);
   });
 
+  test('throw error when description is too long', () => {
```

Comment:
Usually this test name is fine, but in this file, many tests for other errors have names of the following form: `'event bus xx cannot ...'`, so could you please match them?